GitHubCore.ps1

```diff
@@ -210,7 +212,7 @@ function Invoke-GHRestMethod
             if ($PSCmdlet.ShouldProcess($jobName, "Start-Job"))
             {
                 [scriptblock]$scriptBlock = {
-                    param($Url, $method, $Headers, $Body, $TimeoutSec, $ScriptRootPath)
+                    param($Url, $method, $Headers, $Body, $validBodyContainingRequestMethods, $TimeoutSec, $ScriptRootPath)
```

Comment:
It's a bug that `$method` isn't capitalized.  Can you please `PascalCase` `$ValidBodyContainingRequestMethods`?